multisrc/overrides/madara/hentai20/src/Hentai20.kt

```diff
@@ -1,13 +0,0 @@
-package eu.kanade.tachiyomi.extension.en.hentai20
-
-import eu.kanade.tachiyomi.multisrc.madara.Madara
-import eu.kanade.tachiyomi.network.interceptor.rateLimit
-import okhttp3.OkHttpClient
-import java.util.concurrent.TimeUnit
-
-class Hentai20 : Madara("Hentai20", "https://hentai20.io", "en") {
-
-    override val client: OkHttpClient = super.client.newBuilder()
-        .rateLimit(1, 1, TimeUnit.SECONDS)
-        .build()
```

Comment:
Why didn't you preserve the old rate limit?

Reply:
Oh i am sorry. I thought maybe in Mangathemesia it doesn't use it..as i checked perviously.. it was working fine. Let me add that! Better to be careful!

Reply:
> Why didn't you preserve the old rate limit?

Say should i add it in src of hentai20 or ?? Please help me!